govcd/openapi_org_network_dhcp.go

```diff
@@ -95,9 +95,6 @@ func (vdc *Vdc) UpdateOpenApiOrgVdcNetworkDhcp(orgNetworkId string, orgVdcNetwor
 
 // DeleteOpenApiOrgVdcNetworkDhcp allows to perform HTTP DELETE request on DHCP pool configuration for specified Org VDC
 // Network ID
-//
-// Note. VCD Versions before 10.2 do not allow to perform "DELETE" on DHCP pool and will return error. The way to
-// remove DHCP configuration is to recreate Org VDC network itself.
 func (vdc *Vdc) DeleteOpenApiOrgVdcNetworkDhcp(orgNetworkId string) error {
 	endpoint := types.OpenApiPathVersion1_0_0 + types.OpenApiEndpointOrgVdcNetworksDhcp
 	minimumApiVersion, err := vdc.client.checkOpenApiEndpointCompatibility(endpoint)
```

Comment:
While it shouldn't matter for DELETE I vote to use `getOpenApiHighestElevatedVersion` here as well so that all operations are using the same API version.

Reply:
yeah, I agree. I'll change that!

Reply:
Thanks! I was expecting Dainius to have some comments on API version functions used too :)

Reply:
Already changed @Didainius !!! Please check and resolve conversation if you agree 😄